README.md

```diff
@@ -1,4 +1,109 @@
-# lto-api.java
- LegalThings One client for Java
+LTO Network client for PHP
+===
 
-**WIP - Please [comment on this issue](https://github.com/ltonetwork/lto-api.java/issues/5) if you'd like to connect to LTO Network from Java.**
+_Signing and addresses work for the public chain only._
+
+Requirements
+---
+
+- JDK 11 or later
+
+Accounts
+---
+
+### Creation
+
+#### Create an account from seed
+
+```java com.ltonetwork.client.core
```

Comment:
Use `import` rather than working in the lto namespace

Reply:
It was suggested by Intellij... will change it to imports